doc/reference/configuration/configuration_reference.rst

```diff
@@ -3860,6 +4145,78 @@ The ``replication`` section defines configuration parameters related to :ref:`re
     | Default: 0
     | Environment variable: TT_REPLICATION_SYNC_TIMEOUT
 
+
+.. _configuration_reference_replication_synchro_queue_max_size:
+
+.. confval:: replication.synchro_queue_max_size
+
+    **Since:** :doc:`3.3.0 </release/3.3.0>`
+
+    The ``replication.synchro_queue_max_size`` option controls the maximum size of the synchronous transaction queue on a master node.
```

Comment:
Please keep wordings as short as possible. This reference is huge enough.

```suggestion
    The maximum size of the synchronous transaction queue on a master node, in bytes.
```

Reply:
fixed